diem-move/diem-framework/tests/move_verification_test.rs

```diff
@@ -3,17 +3,10 @@
 
 use move_cli::package::prover::ProverTest;
 
+// TODO: split this into individual tests once the package system supports this.
 #[test]
-fn prove_core() {
-    ProverTest::create("core").run()
-}
-
-#[test]
-fn prove_experimental() {
-    ProverTest::create("experimental").run()
-}
-
-#[test]
-fn prove_dpn() {
+fn prove_all() {
```

Comment:
@tzakian It appears a recent change in package system made it impossible to run tests in parallel. This is because the package system modifies the global current dir of the process, and Rust runs different tests in the same process. I have no idea how this pass CI, but it consistently failed locally for me. In general tests should be able to assume they are executed in the crate root dir as current dir.